### ISSUE TITLE
Set headers over SSL

### ISSUE THREAD

**Issue by papattes** [Wisembly/elephant.io#193]
Hello ! 

to set custom headers over an SSL connection, there is a small change to make in the Version1X.php file.

you have to force the protocol used for the headers on 'http' because, according to the documentation, the modification of header can not be done in the SSL context, only in HTTP context.

[https://www.php.net/manual/en/context.ssl.php](https://www.php.net/manual/en/context.ssl.php)
[https://www.php.net/manual/en/context.http.php](https://www.php.net/manual/en/context.http.php)

Line 172 and 173 of Version1X.php file.:

```
if (isset($this->options['headers'])) {
    $headers = isset($context['http']['header']) ? $context['http']['header'] : [];
    $context['http']['header'] = array_merge($headers, $this->options['headers']);
}
```

**Comment by Taluu** [comment]
Please make a PR. :}

**Comment by Taluu** [comment]
Fixed in #194
